openmrs/src/main/java/org/motechproject/openmrs/tasks/builder/ActionBuilder.java

```diff
@@ -147,6 +150,28 @@ private ActionEventRequest buildUpdatePatientAction(String configName) {
                 .createActionEventRequest();
     }
 
+    private ActionEventRequest buildCreateObservationJSON(String configName) {
+        SortedSet<ActionParameterRequest> actionParameters = new TreeSet<>();
+        int order = 0;
+        String serviceMethod = "createObservationJSON";
+
+        actionParameters.add(prepareParameter(Keys.CONFIG_NAME, DisplayNames.CONFIG_NAME, configName, true, true,
+                order++));
+        actionParameters.add(prepareParameter(Keys.OBSERVATION_JSON, DisplayNames.OBSERVATION_JSON, TEXTAREA, "{}", true, order++));
```

Comment:
please create a variable named 'defaultValueForJsonField = "{}". It will be more readable